torch/_refs/__init__.py

```diff
@@ -2200,16 +2206,23 @@ def _make_copy_from_view(fn):
     """
     Given a view function (e.g. torch.diagonal) generates its copy variant (e.g. torch.diagonal_copy)
     """
-    name = fn.__name__
-    fn = out_wrapper()(fn)
+    aten_fn = getattr(aten, fn.__name__)
+    annotations = fn.__annotations__
+    fn = out_wrapper()(aten_fn)
+    fn.__annotations__.update(annotations)
```

Comment:
nit. Move it to after `_fn.__name__` to keep all the metaprogramming together.

Reply:
Done.